src/apps/competitions/tasks.py

```diff
@@ -0,0 +1,11 @@
+from celery import shared_task
+
+from competitions import models
+
+
+@shared_task
```

Comment:
why "shared task?"

Reply:
This is what I am familiar with from Chahub and just what I've used. I'm honestly not sure about task vs shared_task.

Reply:
yeah let's just use task